blocks/marketo/marketo.js

```diff
@@ -28,11 +31,60 @@ const embedMarketoForm = (marketoId, formId, successUrl) => {
           form.onSuccess((values, followUpUrl) => {
             // Take the lead to a different page on successful submit,
             // ignoring the form's configured followUpUrl
-            // eslint-disable-next-line no-restricted-globals
             location.href = successUrl;
+            if (window._satellite) {
+              _satellite.track('formSubmit', {
+                formName: document.title,
+              });
+            }
             // Return false to prevent the submission handler continuing with its own processing
             return false;
           });
+
+          // const conReasonField = document.querySelector('#contactmeHowcanwehelp');
+          // const noButtonReasons = ['Customer Support'];
+          let booleanEvent = true;
```

Comment:
This name is descriptiveon what it is flagging and I would reverse the logic as well.

```
let hasTrackedFormLoad = false;

            if (!hasTrackedFormLoad && window._satellite) {
                window._satellite.track('formLoad', {
                  formName: document.title,
                });
            }

           hasTrackedFormLoad = true;
```

Or move `hasTrackedFormLoad` into `if` block if you want to try on every `whenReady` event until you are successful.

Reply:
Yes. makes sense. I just used the same code they had in their tracking today. Will update